src/server/views/layout.html

```diff
@@ -36,18 +36,19 @@
 
 {% block beforeContent %}
     {% if phaseTag %}
-        {% if feedbackLink %}
+        {% if 'mailto:' in feedbackLink %}
```

Comment:
Ahh from testing, there's another case we've missed

When `feedbackLink` starts with `/` not `http(s)://` we'll need to drop the new tab behaviour